### PR TITLE
feat(sdk)!: rename commands, arguments parameters to command, args

### PR DESCRIPTION
**Pull request by connor-mccarthy**
**Description of your changes:**

Renames `ContainerSpec` and `PipelineTask` parameter names to be consistent with Kubernetes API.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: sdk/python/kfp/components/structures.py [hunk 1]
```diff
@@ -574,10 +574,11 @@ def load_from_component_yaml(cls, component_yaml: str) -> 'ComponentSpec':
         Returns:
             Component spec in the form of V2 ComponentSpec.
         """
+
         json_component = yaml.safe_load(component_yaml)
         try:
             return ComponentSpec.parse_obj(json_component)
-        except pydantic.ValidationError:
+        except (pydantic.ValidationError, AttributeError):
```
Comment (connor-mccarthy) [inline sdk/python/kfp/components/structures.py:581]: @ji-yaqi, I opted to broaden the range of exceptions that will be caught, but still left it more narrow than `except Exception:`. Can you let me know what you think?

Comment (ji-yaqi) [reply sdk/python/kfp/components/structures.py:581]: Yea we could leave it here but I would suspect to have more than attributeError here. I can't think of any for now, and we can add more in the future if needed.

**Comment by ji-yaqi** [comment]
/lgtm
/approve
Thanks @connor-mccarthy!

**Pull request merged**